pyproject.toml

```diff
@@ -113,6 +113,7 @@ module = [
   'SOAPify.*',
   'numba.*',
   'onion_clustering.*',
-  'py.*'
+  'py.*',
```

Comment:
what is py?

Reply:
I really don't know but I don't think I did it (...or maybe yes); it is even present in the main directory. I used this part of code just to add scipy. Maybe @matteobecchi knows something about it and can tell us if is good to remove this line.

Reply:
I don't remember when/why this line was added, but it was probably when we were putting onion into dynsigth.

Reply:
@SimoneMartino98  can you remove this line and see if the tests/mypy/ruff runs ok?